### DISCUSSION TITLE
Disable jdtls notification for Lazyvim

### DISCUSSION THREAD

**Discussion by chetra-seng** [nvim-java/nvim-java#287] [Q&A] [answered]
Hi, I'm trying to disable jdtls notification in Lazyvim. I've also check the [Q & A](https://github.com/nvim-java/nvim-java/wiki/Q-&-A#no_entry-how-to-remove-noisy-and-annoying-jdtls-notifications). However, when I tried this config in LazyVim I got these errors

![Error log](https://github.com/user-attachments/assets/168c3939-3338-4df1-b2db-7b1ed3f7c90b)

Here's my config for Lazyvim

![Config](https://github.com/user-attachments/assets/1f03fa94-9374-43d6-bfd2-f83f72e434e7)

I'm not sure if this happens because I misconfigured lspconfig (skill issues) or due to other causes. It would be great if someone could correct me or explain why these error occurs Thank

**Comment by chetra-seng** [comment, marked as answer]
Update: After tweaking with the configs, the error was not occur due to my configuration. It might got something to do with inlay_hint and jdtls plugins.

**Reply by s1n7ax** [reply]
We don't directly do anything to support inlay_hint. Probably something unrelated or some `jdtls` configuration you have passed.
